tests/dummy/app/templates/docs/between.md

```diff
@@ -0,0 +1,107 @@
+  </tr>
+  <tr>
+    <td class="tg-eh2d">Kept</td>
+    <td class="tg-eh2d">Yes</td>
+    <td class="tg-eh2d">Yes</td>
+  </tr>
+  <tr>
+    <td class="tg-eh2d">Removed</td>
+    <td class="tg-eh2d">Yes</td>
+    <td class="tg-eh2d">No</td>
+  </tr>
+  <tr>
+    <td class="tg-eh2d">Received</td>
+    <td class="tg-eh2d">Remote</td>
+    <td class="tg-eh2d">Local</td>
+  </tr>
+  <tr>
+    <td class="tg-eh2d">Sent</td>
+    <td class="tg-eh2d">Local</td>
+    <td class="tg-eh2d">Remote</td>
+  </tr>
+</table>
+
+
+
+
+### Interruption Cases
+In this demonstration, the "Delete with Undo" option shows what happens when an animation is interrupted. An email that would have been a `removedSprite` becomes a `keptSprite`. 
+
+### Beacons
+You may have noticed that this demonstration uses `#animated-beacon`. For more on beacons, see [animated-beacon](../docs/api/components/animated-beacon). 
+
+In this example, emails animate between Refresh (mail icon), Trash, and the Inbox. When an email gets deleted, it is a `removedSprite`. If you refresh your inbox, the new email added is an `insertedSprite`. This scenario has two beacons, the refresh button and the trash button.
+
+{{#docs-demo as |demo|}}
+    {{#demo.example name="one"}}
+      {{#full-log-table as |fullLog|}}
+        {{logged-between-components fullLog=fullLog}}      
+      {{/full-log-table}}
+    {{/demo.example}}
+
+    {{demo.snippet 'between-components-snippet.hbs' label='between-components.hbs'}}
+    {{demo.snippet 'between-components-snippet.js' label='between-components.js'}}
+    {{demo.snippet 'sprites-snippet.css'}}
+{{/docs-demo}}
+
+### Animating Across Lists
+In this example, the office is hosting a dinner party. Everyone received an email invitation with two options "going" and "not going".  The people invited can change their response as many times as they want. If Dwight said he was going then changes his mind, he will be removed from the "going" list and added to the "not going" list. This means that Dwight is a `sentSprite` on the going list and a `receivedSprite` on the "not going" list. If Dwight changes his mind again, he would be removed from the "not going" list and added to the "going" list. In this case, Dwight would be a `sentSprite` on the "not going" list and a `receivedSprite` on the "going list". 
+
+
+{{#docs-demo as |demo|}}
+    {{#demo.example name="two"}}
+      {{#full-log-table as |fullLog|}}
+        {{logged-two-lists fullLog=fullLog}}
+      {{/full-log-table}}
+    {{/demo.example}}
+
+    {{demo.snippet 'between-two-lists-example-snippet.hbs' label='between-two-lists-example.hbs'}}
+    {{demo.snippet 'between-two-lists-example-snippet.js' label='between-two-lists-example.js'}}
+    {{demo.snippet 'two-lists-snippet.css'}}
+{{/docs-demo}}
+
+
+### Animating Across Routes
+This is an example of animating sprites across different routes. When you select an icon from the list, the selected image and the list of images animate as the route changes. 
```

Comment:
Need a lead in sentence. "Just like animations can be applied to components, they can be applied to route transitions too!"